config/clients/java/template/.github/workflows/main.yml.mustache

```diff
@@ -0,0 +1,27 @@
+name: Java CI with Gradle
```

Comment:
TODO: This is CI for the `fga-java-sdk` project. I should add CI for _this_ `sdk-generator` project, too.

Reply:
Moved this to a checkbox in the PR description